livekit_sip.proto

```diff
@@ -151,10 +143,6 @@ message SIPDispatchRule {
     // enter the room
     SIPDispatchRuleDirect dispatch_rule_direct = 1;
 
-    // SIPDispatchRulePin is a `SIP Dispatch Rule` that allows a user to choose between multiple rooms.
```

Comment:
@dennwc I think you need to update docs in a few other places.

Mind also updating the id so it is `1 -> 2` since we aren't public yet, should be ok?

Reply:
Done!